stable/spark/docker_images/zepplin/zeppelin-env.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# export JAVA_HOME=
```

Comment:
There is an awful lot of commented out code in this file. What is the intention of this?

Reply:
I think Dalitso might have included this because it shows new users what are the possible nobs and dials they can turn

Reply:
You might want to re-package this as a ConfigMap.

Reply:
Yes @mhamilton723 . The intention was to show users the environment variables they can play around with. In our setup we change the spark master of zeppelin to point to the master on kubernetes. @wirehead this is a a file for the zeppelin environment variables only rather than a cluster wide env vars that I think would be more suited for config maps.